mixin/prometheus_alerts.yaml

```diff
@@ -16,7 +16,7 @@ groups:
             description: Compute credit usage is {{ printf "%.2f" $value }} credits/hr for {{$labels.instance}}, which is within 20% of 5 credits/hr.
             summary: Compute credit usage is within 20% of the configured limit.
           expr: |
-            sum by (job, instance) (last_over_time(snowflake_used_compute_credits{}[1h]))
+            sum by (job, instance) (last_over_time(snowflake_used_compute_credits{}[24h]))
```

Comment:
Please revert this change, it's not in line with the description or other alerts